canonical-web-and-design/snapcraft.io
# Update pattern names

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1093

## Done
- Replace `p-list-step` with `p-stepped-list` including updated to child nodes
- Replace `p-code-snippet` with `p-code-copyable` including removal of most of the custom snapcraft css
- Replaced `<code><pre>` with `<pre>` in fsf
- Added `u-no-limit` for overriding `max-width` headings

## QA
- Pull the branch
- Ensure the SCSS builds
